docs/source/daml/intro/3_Data.rst

```diff
@@ -0,0 +1,202 @@
+Data Types
+==========
+
+Contract templates specify the types of contracts that can be created on the ledger, and what data they hold in their arguments (see :doc:`1_Token`). The scenario view in the IDE displays the current ledger state by showing one table per template, with one row per contract of that type and one column per field in the arguments. This analogy is a useful one. Templates specify a data schema for the ledger, with each template corresponding to a table, each field in the ``with`` block of a template corresponding to a column in that table, and each contract instance of that type corresponding to a table row.
+
+In this section, you'll learn how to create rich data schemas for your ledger. Specifically you'll learn about:
+
+- DAML's built-in and *native* data types
+- Record types
+- Derivation of standard properties
+- Variants
+- Manipulating immutable data
+- Contract keys
+
+After this section, you should be able to use a DAML ledger as a simple database where individual parties can write, read and delete complex data.
+
+Native Types
+------------
+
+You have already encountered the type ``Party`` in :doc:`1_Token` and ``Text`` and ``ContractId`` in :doc:`2_Scenario`.
+
+- ``Party``
+  Stores the identity of an entity that is able to act on the ledger.
+- ``Text``
+  Stores a unicode character string like ``"Alice"``.
+- ``ContractId a``
+  Stores a reference to a contract of type ``a``.
+- ``Int``
+  Stores unsigned 64-bit integers. For example, ``-123``.
+- ``Decimal``
+  Stores fixed-point number with precision 38 and scale 10. For example, ``0.0000000001`` or ``-9999999999999999999999999999.9999999999``.
```

Comment:
If we expect to appeal to simple-minded users like me, I think it's worth adding
```suggestion
  Stores fixed-point number with precision 38 and scale 10, i.e. 28 digits before and 10 digits after the decimal point. For example, ``0.0000000001`` or ``-9999999999999999999999999999.9999999999``.
```